### DISCUSSION TITLE
Is it possible to avoid HTML blocks around references in `format: gfm`?

### DISCUSSION THREAD

**Discussion by pat-alt** [quarto-dev/quarto-cli#408] [Q&A] [answered]
Is it possible to avoid HTML blocks around references in `format: gfm`? Or any other markdown format? I basically just want the reference block rendered without need for actual cross references from citation to reference.

**Background**: I'm working on a Julia package [CounterfactualExplanations.jl](https://github.com/pat-alt/CounterfactualExplanations.jl) and am trying to use Quarto to create docs. I basically first write the docs including code in Jupyter, render to `format: gfm` and then reference the rendered .md files in my make.jl file. The problem is that [Documenter.jl](https://juliadocs.github.io/Documenter.jl/stable/) doesn't play nicely with raw HTML (one needs to wrap it in blocks). Apparently there are more Julian ways to deal with such things, but I love Quarto and want to use it where I can. 

Any thoughts would be much appreciated!

**Comment by jjallaire** [comment, marked as answer]
That seems to the underlying behavior of pandoc for gfm (because gfm has raw_html enabled by default). Unfortunately disabling `raw_html` seems to still result in the same output. However, using `commonmark` w/o `raw_html` seems to work:

```yaml
format: 
  commonmark:
    variant: -raw_html
```

**Reply by pat-alt** [reply]
That worked! Thanks very much 😄

**Comment by jjallaire** [comment]
Hi there @pat-alt, posting back here because I'm giving a talk on Quarto at JuliaCon next month and was hoping to find some good examples of using Julia w/ Quarto. If you are able to share any of your own (or are aware of others) please LMK over on this discussion thread: https://github.com/quarto-dev/quarto-cli/discussions/1261. Thanks!

**Reply by pat-alt** [reply]
Hey @jjallaire 😁 That's so cool that you'll be at JuliaCon! Funnily enough I'll also be giving an Experience Talk about my experience with using Julia + Quarto (e.g for building package docs). Shall we perhaps also coordinate offline to avoid overlaps? Either way I'll share some stuff tomorrow on the discussion you linked 🤟

**Reply by jjallaire** [reply]
Yes by all means let's coordinate. I'm working on my outline today and tomorrow and will send you a DM on twitter when its shaped up.

**Reply by pat-alt** [reply]
For reference: my [repo](https://github.com/pat-alt/QuartoJulia) and the [presentation](http://www.paltmeyer.com/QuartoJulia/#/title-slide) it builds. This is a very short talk and I'll breeze through the first slide and just mention that your talk will cover more detail. Do feel free to reach out though or let me know if there's a clear overlap somewhere.
